UKHomeOffice/cerberus-service
# Cop 9564 hide sections

## Description
This PR hides and displays vehicle/ trailer sections for RORO Unaccompanied mode.
- JIRA Ticket: https://support.cop.homeoffice.gov.uk/browse/COP-9564

## To Test
- Pull & run against dev.
- Using the filter, show a list of **RORO Unaccompanied** Tasks.
- Select a task and view its details.
> Verify that:
1) For RoRo Unaccompanied task(s), the vehicle section is not rendered.
2) For RoRo Unaccompanied task(s), the trailer section is rendered.


## Developer Checklist

\* Required
